src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,7 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+
```

Comment:
The method `getSalaryInfo` is not yet implemented. You need to add logic to calculate the salary information based on the provided names, data, and date range. Make sure to parse the dates correctly and sum up the salaries for each name within the specified date range.